libr/arch/p/arm/plugin_cs.c

```diff
@@ -2507,7 +2507,7 @@ static int analop_esil(RArchSession *as, RAnalOp *op, ut64 addr, const ut8 *buf,
 			if (ISIMM (0)) {
 				r_strbuf_appendf (&op->esil, "%s,pc,:=", ARG (0));
 			} else {
-				r_strbuf_appendf (&op->esil, "%d,%s,-,pc,:=", thumb, ARG (0));
+				r_strbuf_appendf (&op->esil, "%s,pc,:=,-1,1,<<,pc,&", ARG (0));
```

Comment:
i think it should be this way
```suggestion
				r_strbuf_appendf (&op->esil, "-1,1,<<,%s,&=", ARG (0));
```